pyproject.toml

```diff
@@ -45,10 +44,10 @@ include = [
 
 [tool.poetry.dependencies]
 
-python = ">=3.6.1,<3.10"
+python = ">=3.7.0,<3.10"
 
 awscli = ">=1.18.174"
-dcicutils = ">=3.9.0"
+dcicutils = ">=3.9.0,<4"
```

Comment:
Is capping the version here done out of precaution or are there incompatibilities that need to be addressed?